<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MOutputDialog</class>
 <widget class="QDialog" name="MOutputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="function">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>241</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>导出functions</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="image">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>导出预览图片</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="schematic">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>导出schematic(暂不可用)</string>
   </property>
  </widget>
  <widget class="QToolButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="path">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>dir</string>
   </property>
  </widget>
  <widget class="QToolButton" name="output">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>开始导出</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>380</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>文件命名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
